base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -138,22 +138,22 @@ impl ConsensusConstants {
 
```

Comment:
```suggestion
// Adjusting the block window N is based on the following:
https://github.com/zawy12/difficulty-algorithms/issues/14
```

I think it would be good to add this here for future reference on why the block window is what it is.